Integrations/integration-VxStream.yml

```diff
@@ -428,3 +459,19 @@ script:
       description: 'environment ID to submit file to(get all via vx-get-environments) '
     description: Retrieve result data upon a file. NOTE - This command returns a file
       (also - minimum required authorization is `default`)
+  - name: vx-detonate-file
+    arguments:
+    - name: entryId
+      required: true
+      description: War-room entry ID of sample file
+    - name: environmentID
+      required: true
+      description: 'environment ID to submit file to(get all via vx-get-environments) '
```

Comment:
Change this:
`'environment ID to submit file to(get all via vx-get-environments) '`
to
`Environment ID to submit file to (get all IDs via vx-get-environments)`